src/pages/Home.jsx

```diff
@@ -0,0 +1,27 @@
+import { useNavigate, Link } from 'react-router-dom';
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+
+const Home = (props) => {
+  let navigate = useNavigate();
+  const { token, setToken } = props;
+
+  const generateToken = () => {
+    const token = getToken();
+    localStorage.token = token;
```

Comment:
I actually did not realize you could do this. [Documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) shows using `getItem` and `setItem`. I would recommend that approach in the future because it's more flexible (you can use any string as the key, like strings with spaces, etc.) and I think it's going to be more familiar to people.

Reply:
Thanks Rob! I am going to circle back to this during the week.